### ISSUE TITLE
A11y_Oracle Assessment for Azure Data Studio (ADS)_AssessmentReport_FeatureCompatibility_ScreenReader: Screen Reader is not announcing the message information about the grid item after pressing enter.

### ISSUE THREAD

**Issue by v-akr** [microsoft/azuredatastudio#19581]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win11;#DesktopApp;#WCAG4.2.1;#Benchmark;#FTP;#NVDA;#JAWS;

### Environment Details:
Application Name: Oracle Assessment for Azure Data Studio (ADS)
Operating System: Windows 11
Version :1.35.1
Screen reader: NVDA(2021.3.5)

### Steps to Reproduce:

1. Download the azure data studio
2. Open the application and tab to the Connections icon in left navigation.
3. Tab to the new connections button and press enter. Connection dialog box will open.
4. Fill Correct details in connection details and tab to the connect button and press enter. New connection will be added.
5. Tab to the Database Migration Assessment for Oracle button and then press enter. "Database Migration Assessment for Oracle" page will be opened.
6. Tab to the "Run New Assessment" button at top the page and press enter.New Database Migration Assessment for Oracle dialog box will open. Fill the correct details. Tab to "Start Assessment button and press enter.
7. New Assessment can be seen in last five assessment tab. Tab to the assessment press enter. New tab will get open.
8. Tab to Assessment overview by platforms section. Tab to Assessment for Azure virtual machine in the section.Tab to view report details button and press enter. New screen will open.
9. Tab to "Feature Compatibility" tab and press enter.
10. Start NVDA and observe the data item in the grid. 

### Actual:

Screen Reader is not announcing the message information about the grid item after pressing enter.

**Similar issue is observed for Jaws also.**

### Expected:
Screen Reader should be announcing the message information about the grid item after pressing enter.

### User Impact:
Screen reader user will not be going to know about the message for the selected grid item.
